python/nano/src/bigdl/nano/utils/pytorch/model_info.py

```diff
@@ -0,0 +1,135 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import inspect
+from typing import Optional, Union
+
+import torch
+from bigdl.nano.pytorch.model import AcceleratedLightningModule
```

Comment:
Don't import `AcceleratedLightningModule` here.
Because `bigdl.nano.pytorch.xxxx` will import `bigdl.nano.utils.pytorch.xxxx`, if you import `bigdl.nano.pytorch.xxxx` here and add this class to `nano/utils/python/__init__.py`, it may cause circular references, then import recursively.
You can import `AcceleratedLightningModule` inside thd function body.